.github/workflows/build_for_deployment.yaml

```diff
@@ -0,0 +1,50 @@
+name: mara-app
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+jobs:
+  mara-app-build:
```

Comment:
For C&P reasons, this would be nice to have a generic name :-)

Reply:
you mean simple "build" and in the other step "deploy"? sounds good for me.